packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+| title         | `string`                                 | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                | `true`      | Shows a dismiss button                                                                     |          |
+| contrast      | `high`, `low`                            | `low`       | Can be set to `high` for more prominent look _(not related to a11y)_                       |          |
+| isFullWidth   | `boolean`                                | `false`     | Spans the entire width of container, otherwise max width is restricted to 584px by default |          |
+| variant       | `info`, `positive`, `notice`, `negative` | `info`      | Sets the color tone of entire alert. Icon is set automatically based on variant \*         |          |
+| isBorderless  | `boolean`                                | `false`     | Removes borders and border radii, useful for creating a full bleed layout                  |          |
+
+We'll also expose the following sub components on `Alert` which can be passed as children to `Alert`. The order of passing these as children doesn't matter:
+
+| Sub component           | Description                                                          | Required |
+| ----------------------- | -------------------------------------------------------------------- | -------- |
+| `Alert.Description`     | For setting body content                                             | ✅       |
+| `Alert.PrimaryAction`   | Renders a primary `Button`                                           |          |
+| `Alert.SecondaryAction` | Renders a button `Link`, requires `Alert.PrimaryAction` alongside \* |          |
+
+### Alert.Description
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content \*  | ✅       |
+
+### Alert.PrimaryAction
+
+`Alert.PrimaryAction` renders and accepts a subset of `Button` (primary) props:
+
+| Prop       | Type       | Default     | Description                            | Required |
+| ---------- | ---------- | ----------- | -------------------------------------- | -------- |
+| `children` | `string`   | `undefined` | Content, required here unlike `Button` | ✅       |
+| `onClick`  | `function` | `undefined` | Callback                               | ✅       |
+
+### Alert.SecondaryAction
+
+`Alert.SecondaryAction` renders and accepts a subset of `Link` (button) props:
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content     | ✅       |
+| `href`     | `string`      | `undefined` | Hyperlink   |          |
+| `onClick`  | `function`    | `undefined` | Callback    |          |
```

Comment:
A simple note here should be that `href` and `onClick` are mutually exclusive and cannot be used together.

Reply:
These can be used together (href is good for SEO reasons and onClick for handling actions). The secondary action will render a link button:
- [Decision docs for link button](https://github.com/razorpay/blade/blob/docs/link-decisions/packages/blade/src/components/Link/_decisions/decisions.md#link-component)
- [Related thread on Link component doc PR](https://github.com/razorpay/blade/pull/538#discussion_r912915149)